lms/djangoapps/course_api/serializers.py

```diff
@@ -14,9 +14,9 @@ class _MediaSerializer(serializers.Serializer):  # pylint: disable=abstract-meth
     """
     Nested serializer to represent a media object.
     """
-    def __init__(self, uri_attribute, *args, **kwargs):
+    def __init__(self, *args, **kwargs):
+        self.uri_attribute = kwargs.pop("uri_attribute", None)
```

Comment:
This was a fix to a bug that was preventing Swagger from generating documentation. If you're able to see swagger docs on your version, then it no longer applies, which would be great.